### PR TITLE
feat: new ContainerV2 component

### PR DESCRIPTION
**Pull request by Exayy**
## Type

- Bug
- Feature ✅ 
- Enhancement
- Documentation
- Migration
- Refactor

### Summarise concisely:

New component : ContainerV2, which is the same component than Container (v1) without Box props and margin-top

### CODE REVIEW

File: src/components/ContainerV2/index.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <Stack gap={1} direction="row" alignItems="center">
+          <Text variant="heading" as="h2">
+            {title}
+          </Text>
+          {subtitle}
+        </Stack>
+        <div>{rightTitle}</div>
+      </Stack>
+    ) : null}
+    {header}
```
Comment (chambo-e) [inline src/components/ContainerV2/index.tsx:65]: header seems to be used only once which could be replaced by subtitle

Maybe nuke this props ?

**Review by matthprost — CHANGES_REQUESTED** [review]
Can you remove loki update as we are using chromatic ?

File: src/components/ContainerV2/index.tsx (hunk 1 above)
Comment (Exayy) [reply src/components/ContainerV2/index.tsx:65]: actually `subtitle` hasn't the same positionning it's next (on the right) to the title, header is below

Props are really confusing :
- Title & subtitle are close, mostly subtitle has another font style
- right title sticks to the right
- header : below everything

`header` isn't bad IMO even if it's use only once so far, i've more doubt about `subtitle`

**Review by chambo-e — APPROVED** [review]

**Review by Lawndlwd — APPROVED** [review]
💜

**Pull request merged**